actions/lib/actions.py

```diff
@@ -33,6 +33,9 @@ def __init__(self, config):
         except KeyError:
             raise ValueError("api_key needs to be configured!")
 
+        if self.config.get("region") == "eu":
```

Comment:
Don't make this a special case here - that just creates an additional maintenance burden. Instead, simply conditionally pull the region from the config.

```diff
-         if self.config.get("region") == "eu":
-             self.API_HOST = "https://api.eu.opsgenie.com/"
+        if self.config.get("region"):
+            self.API_HOST = "https://api.{region}.opsgenie.com/".format(region=self.config.get("region"))
```

Reply:
Cheers, I've addressed this.